### DISCUSSION TITLE
Upgrade questions

### DISCUSSION THREAD

**Discussion by hakha4** [fredlcore/BSB-LAN#675] [General]
Hi!
I've used BSB-LAN ver 2 since 2021 and are very satisfied with the functionality except one issue. As stated in documentation DS1820 sensors can loose order and that what troubles me so I plan to upgrade to ver4 and include the script that fix this problem. In documentation it says that #define CUSTOM_COMMANDS shall be enabled but I cant find this anywhere in code. In what file shall I put this, BSB_LAN _custom_globals.h ? 
Best Regards Håkan

**Comment by fredlcore** [comment]
It's there where everything else is also configured, in `BSB_LAN_config.h`.
